Sources/UIKitExtensions/UISlider+FunctionConfigurationSet.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  UISliderFunctions.swift
+//  ViewConfigurator-iOS
+//
+//  Created by Andrii Pazynych on 27.03.18.
+//  Copyright © 2018 ImagineOn GmbH. All rights reserved.
+//
+
+
+import UIKit
+public extension ConfigurationSet where Base: UISlider {
+    
+    func setValue(_ value: Float, animated: Bool) -> ConfigurationSet<Base> {
+        return set { ( UISlider: Base ) in
+            UISlider.setValue(value, animated: animated)
```

Comment:
This function is only useful for existing UISlider whose value change should be animated.
The value Property ist writable directly so there is no need for this function.